src/displayapp/screens/settings/Settings.cpp

```diff
@@ -58,5 +61,15 @@ std::unique_ptr<Screen> Settings::CreateScreen2() {
     {Symbols::list, "About", Apps::SysInfo},
   }};
 
-  return std::make_unique<Screens::List>(1, 2, app, settingsController, applications);
+  return std::make_unique<Screens::List>(1, 3, app, settingsController, applications);
+}
+
+std::unique_ptr<Screen> Settings::CreateScreen3() {
+
+  std::array<Screens::List::Applications, 4> applications {{
+    {Symbols::clock, "Set date", Apps::SettingSetDate},
+    {Symbols::clock, "Set time", Apps::SettingSetTime},
```

Comment:
Maybe the actual settings should be grouped together on top. So raise this to at least below Steps in the list?

Reply:
Sorry, I don't understand what you mean by grouping them together on top. Should I just put the two entries on the second page and move the succeeding entries to the third page? Or is there a way to actually group the two into a single entry?

Reply:
> Should I just put the two entries on the second page and move the succeeding entries to the third page?

Yes. I think that order would make more sense. It might also make sense to group them into a single entry, but that's a separate thing to think about.

Reply:
Done. I've moved both entries just between Steps and Battery on the second page. Firmware and About dropped to the third page.